pumpp/task/base.py

```diff
@@ -128,6 +128,9 @@ def transform(self, jam, query=None):
         # Prefix and collect
         return self.merge(results)
 
+    def transform_annotation(self, ann, duration):
+        raise NotImplementedError
+
```

Comment:
There shouldn't be any new code in the base class, i think?

Reply:
You are right. I was trying to understand how things are working when I was playing around with this.
line 131-133 should go.

Reply:
Per our offline discussion:
this function `transform_annotation` is called in the `BaseTransformer.transform()` method, and should be stubbed here. So these lines stay and new unit test to cover it.